src/components/screens/error-page.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement } from "react";
+import { useRouteError } from "react-router-dom";
+
+/**
+ * Error page for displaying error status text and message
+ * 
+ * @returns ReactElement
+ */
+export default function ErrorPage (): ReactElement {
+  const error: unknown = useRouteError();
```

Comment:
Does this come typed as unknown by default? I would usually encourage you to find a better type if possible but it this component is likely to change in the future lets not worry about it for now.

Reply:
Yes, it's the default. We will look into better error handling.